pykechain/models/part2.py

```diff
@@ -228,13 +228,28 @@ def populate_descendants(self, batch: int = 200) -> Union['Partset', List['Part2
         >>> bike.populate_descendants(batch=150)
 
         """
-        descendants_flat_list = list(self._client.parts(
```

Comment:
we need to add `.. versionchanged:: 3.3 <bla bla>` as the operation of this method changed

Reply:
I added a string notifying the update. There is no (as far as I know) change in external perspective of this method.